tensorflow_addons/image/solarize_ops.py

```diff
@@ -0,0 +1,40 @@
+""" This module is used to invert all pixel values above a threshold
```

Comment:
Thanks @vinayvr11. Can you also add license like this? https://github.com/tensorflow/addons/blob/4eeea6f972b18aff7df60202c7386b35a455a832/tools/releases/release_linux.sh#L2